### PR TITLE
Istio 1.4 istioctl

### PR DESCRIPTION
**Pull request by piotrmsc**
**Description**

Changes proposed in this pull request:

- upgrade Istio to version 1.4.2
- install Istio using manifest file generated with istioctl instead of using deprecated Helm Charts
- update knative-serving to version v0.11.1
- fix graphana service name
- fix jaeger virtualservice to comply with validation of Istio CRs

**Related issue(s)**

### CODE REVIEW

**Comment by anishj0shi** [comment]
in this document: https://kyma-project.io/docs/components/tracing#troubleshooting-basic-troubleshooting
we are referencing an obsolete istio property in istio-pilot **traceSampling**, this is now influenced via a environment variable: ** PILOT_TRACE_SAMPLING**

**Comment by kubadz** [comment]
> in this document: https://kyma-project.io/docs/components/tracing#troubleshooting-basic-troubleshooting
> we are referencing an obsolete istio property in istio-pilot **traceSampling**, this is now influenced via a environment variable: **PILOT_TRACE_SAMPLING**

We will create another PR for changes in documentation. After it is created I will link it here.

File: resources/jaeger/templates/vs-job.yaml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
```
Comment (kubadz) [inline resources/jaeger/templates/vs-job.yaml]:
```suggestion
---
# these resources can be removed after the release of Kyma 1.10; this job is necessary to upgrade Kyma from version 1.9 to 1.10
```

**Pull request merged**